source/core/TypeAndDataManager.cpp

```diff
@@ -67,8 +67,9 @@ TypeManager::TypeManager(TypeManagerRef parentTm)
     _totalAQAllocated = 0;
     _totalAllocationFailures = 0;
     _maxAllocated = 0;
-    _allocationLimit = 16777216;  //16 meg for starters
-    
+    //_allocationLimit = 16777216;  //16 meg for starters
+    _allocationLimit = 524288000;  //500 meg
```

Comment:
This is a good time to make it configurable.  A fixed amount this large will make LV web pages a memory hog.